get-latest-tgf.sh

```diff
@@ -45,10 +63,10 @@ install_latest_tgf () {
 get_local_tgf_version
 get_latest_tgf_version
 
-echo '- tgf version (local):' $TGF_LOCAL_VERSION
-echo '- tgf version (latest):' $TGF_LATEST_VERSION
+echo '- tgf version (local) :' "${TGF_LOCAL_VERSION}"
+echo '- tgf version (latest):' "${TGF_LATEST_VERSION}"
 
-if [[ $TGF_LOCAL_VERSION == $TGF_LATEST_VERSION ]]
+if [[ ${TGF_LOCAL_VERSION} == "${TGF_LATEST_VERSION}" ]]
```

Comment:
I can more reliably tell what's going to happen when both are wrapped in strings. That may just be me though. I've been bitten by different types in bash and now I wrap everything in strings :smile:

Reply:
According to the intellij shell plugin, it's not necessary, but I prefer quoting both too!  :)
Done: ca5c1db